brainstormforce/astra
# AST-1473 - Applying content-background option to backgrounds of full with (contained+stertched) layouts

### Description
- Need to work Content Background color as background-color for full-width content layout (contained + stretched) cases.
- Right now, Site background color is appearing as background-color for Full width content layouts - https://share.bsf.io/DOu60DE9
- But Content Background color should apply here - https://share.bsf.io/nOu5ZoLP
```
add_filter( 'astra_get_option_apply-content-background-fullwidth-layouts', '__return_true' );
```

### Screenshots
- Customizer where content-bg applying to full-width contained layout - https://share.bsf.io/kpu6joNx
- Frontend view - https://share.bsf.io/12uP26o9
- Editor view - https://share.bsf.io/2NuPNREl

### Types of changes
- New improvement

### How has this been tested?
- Check with every site-layout
- Check special WP pages like 404, search, author, archives etc with assigned layouts 
- Check customizer preview, frontend view, editor view
- Check with background, spacings option

### Checklist:
